eventfiles/form.py

```diff
@@ -0,0 +1,18 @@
+from .models import EventFile
+from django.forms import ModelForm
+from events.models import EventParticipant
+
+
+class MyEventFileCreationForm(ModelForm):
+    class Meta:
+        model = EventFile
+        fields = '__all__'
+
+    def save(self, commit=True):
+        data = self.data.copy()
+        data['participant_id'] = EventParticipant.objects.get(id=1)
+        self.data = data
+        instance = super(MyEventFileCreationForm, self).save(commit=False)
+        if commit:
+            instance.save()
+        return instance
```

Comment:
You don't have to override the save method if the only thing you are going to do is change the participant id.
you can exclude the participant id from the form itself -> exclude means that it won't be cleaned
and then you can save with commit false and change the instance in the view.

An Example of that can be found in the reminder form PR:
Form:
https://github.com/redhat-beyond/lets-meet/blob/04f6afac8f75c3ca2d24a701b9f2aee9feed885b/reminders/forms.py#L10-L18

View:
https://github.com/redhat-beyond/lets-meet/blob/04f6afac8f75c3ca2d24a701b9f2aee9feed885b/events/views.py#L30-L32

Reply:
Done, thanks 😄